### DISCUSSION TITLE
Source breaking changes from `Reapply keyword params`

### DISCUSSION THREAD

**Discussion by xtqqczze** [dotnet/runtime#110216] [General]
There are source breaking changes in https://github.com/PowerShell/PowerShell/pull/24621 from moving to `<LangVersion>13.0</LangVersion>`, likely related to https://github.com/dotnet/runtime/pull/101499. That PR is labelled `needs-breaking-change-doc-created`, has there been a breaking change doc published?

cc: @jozkee @stephentoub

**Comment by jozkee** [comment]
> has there been a breaking change doc published? 

I don't think so. If you could post a repro of the breaking change you are observing, it may help ensure it gets addressed when the doc is created.

@jeffhandley @dotnet/area-system-io for visibility.

**Reply by xtqqczze** [reply]
Repro is here: https://github.com/dotnet/aspnetcore/pull/55010#issuecomment-2059637820

```log
error CS0121: The call is ambiguous between the following methods or properties: 'Program.Join(string, params string[])' and 'Program.Join(string, params ReadOnlySpan<string>)'
```

Since https://github.com/dotnet/runtime/pull/101499 has label `needs-breaking-change-doc-created` it appears there is still no documentation for this issue.

**Comment by xtqqczze** [comment]
Opened issue: https://github.com/dotnet/docs/issues/44102
